src/main/car/inventory/Vehicle.java

```diff
@@ -15,23 +16,28 @@ public class Vehicle {
 	private String vehicleModel;
 	@JsonProperty("vehicle_manufacturer")
 	private String vehicleManufacturer;
+	@JsonProperty("acquisition_date")
+	private Date acquisitionDate;
 
 	/*
-		the price value may need to be changed to Java's Currency. Or it could be an Enum or two values.
+	 * the price value may need to be changed to Java's Currency. Or it could be an
```

Comment:
you can remove these placeholder comments I added since you've added these new fields. They don't apply anymore.